eng/pipelines/runtimelab.yml

```
@@ -57,22 +57,27 @@ stages:
   #
   - template: /eng/pipelines/common/checkout-job.yml
 
-#
-# Build with Release config and Release runtimeConfiguration
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/common/global-build-job.yml
-    buildConfig: Release
-    platforms:
-    - Linux_x64
-    - FreeBSD_x64
-    jobParameters:
-      testGroup: innerloop
-      buildArgs: -s clr+libs+libs.tests+installer -c $(_BuildConfig) /p:ArchiveTests=true
-      extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
-      extraStepsParameters:
-        uploadTests: true
+  #
+  # Build with Release config and Release runtimeConfiguration
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/common/global-build-job.yml
+      buildConfig: Release
+      platforms:
+      - Linux_x64
+      - FreeBSD_x64
+      jobParameters:
+        isOfficialBuild: ${{ variables.isOfficialBuild }}
+        testGroup: innerloop
+        extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
+        extraStepsParameters:
+          uploadLibrariesTests: ${{ eq(variables.isOfficialBuild, false) }}
+          uploadUnsignedArtifacts: false
```

You need to change this to be the value of isOfficialBuild as we want to ship the packages produced here for freeBSD right?

```suggestion
          uploadUnsignedArtifacts: ${{ variables.isOfficialBuild }}
```

---

updated.